src/Symfony/Component/Messenger/Bridge/Doctrine/Transport/Connection.php

```diff
@@ -542,9 +542,9 @@ private function addTableToSchema(Schema $schema): void
 
         // We need to create a sequence for Oracle and set the id column to get the correct nextval
         if ($this->driverConnection->getDatabasePlatform() instanceof OraclePlatform) {
-            $idColumn->setDefault('seq_'.$this->configuration['table_name'].'.nextval');
+            $idColumn->setDefault($this->configuration['table_name'].'_seq'.'.nextval');
```

Comment:
```suggestion
            $idColumn->setDefault($this->configuration['table_name'].'_seq.nextval');
```

Reply:
@alexandre-daubois : I updated that portion of code for consistency.
@rjd22 : Is this if block really needed as messenger in it's auto config process already create a sequence and trigger in case of id not specified ?

Reply:
Yes, for Oracle installations the default block is needed. Else the field won't use the sequence on some installations.